teemaderegister-be/src/controllers/admin.js

```diff
@@ -44,3 +45,39 @@ module.exports.getSecret = async (req, res) => {
 
   return res.json({_id})
 }
+
+module.exports.getCurriculums = async (req, res) => {
+  let curriculums = await Curriculum.find({})
+
+  for (const key in curriculums) {
```

Comment:
instead of this You can use populate (like SQL join) 
```
  const curriculums = await Curriculum.find({}).populate('representative', '_id profile')

```

Then You have this data available in UI
<img width="932" alt="Screenshot 2021-01-13 at 23 13 10" src="https://user-images.githubusercontent.com/4492641/104511175-02f6a880-55f5-11eb-99d4-dce6f92612a9.png">